examples/compose/README.md

```diff
@@ -150,11 +150,8 @@ vitess/examples/compose$ ./lvtctl.sh Help
 - vtgate web ui:
   http://localhost:15099/debug/status
 
-- vtorc web ui:
+- vtorc ui:
   http://localhost:13000
-
-- vtorc debug ui:
-  http://localhost:13200
```

Comment:
We no longer have the old UI. Website docs have to be changed for this. Same change has been made for all the other examples as well.